spec/graphql/mutations/invoices/finalize_spec.rb

```diff
@@ -44,4 +45,35 @@
       end
     end
   end
+
+  context 'with tax provider' do
+    let(:integration) { create(:anrok_integration, organization:) }
+    let(:integration_customer) { create(:anrok_customer, integration:, customer:) }
+
+    before do
+      integration_customer
+    end
+
+    it 'returns pending invoice' do
+      freeze_time do
+        result = execute_graphql(
+          current_user: membership.user,
+          current_organization: organization,
+          permissions: required_permission,
+          query: mutation,
+          variables: {
+            input: {id: invoice.id}
+          }
+        )
+
+        result_data = result['data']['finalizeInvoice']
+
+        aggregate_failures do
```

Comment:
we aggregate_failures by default, hence, this is not needed anymore.

Reply:
updated